tests/parser/test_term.py

```diff
@@ -0,0 +1,83 @@
+from ats.parser.parser import parse
```

Comment:
@Monikaa1909, @patrycjarecko: po co w ogole powtarzac te testy? Jezeli to wszystko nie jest juz w obrebie expression, to przeniescie testy z expression do term.

Dodatkowo brakuje testow odnosnie takich przypadkow jak `a + b * c` czy `a  * b + c`

Reply:
@wvffle są różne testy dla plusów i minusów i one są w test_expr.py 
a mnożenie jest w test_term.py (tam są dwa duplikaty ale już ogarnęłam)

a + b * c czy a  * b + c dodać do nowego pliku testowego skoro to jest combo expr i term?

Reply:
z tego co pamietam, to term jest skladnikiem expr, wiec dodac w expr